client/data/disputes/test/resolvers.js

```diff
@@ -68,6 +69,9 @@ describe( 'getDispute resolver', () => {
 					expect.any( String )
 				)
 			);
+			expect( generator.next().value ).toEqual(
+				updateErrorForDispute( 'dp_mock1', undefined, errorResponse )
```

Comment:
Curious what regressions tests like this would catch.

Reply:
I guess it could help ensure that we’re calling the update function in the case of a refactor but it feels a bit like we’re testing implementation details here.

Note that this isn’t a new test, I added this here to fix a failing test after adding updateErrorForDispute to the getDispute resolver.

Reply:
Gotcha, figured there would be a thread pulling you here 😁